src/main/java/org/elasticsearch/cluster/ClusterState.java

```diff
@@ -167,6 +176,10 @@ public long getVersion() {
         return version();
     }
 
+    public String uuid() {
```

Comment:
can we have a simple doc string telling what this UUID gives us and where it comes from?